Garronte el búho

Jinouga19 opened the pull request:
## What Does This PR Do
Añade un nuevo buhó en la oficina del chaplain, con los ojos muy grandes con los cuales cuidara la iglesia cuando no estes con su particular sonido.
Añade una cama para la nueva mascota.

## Why It's Good For The Game
Me parece que el trabajo de chaplain es un poco solitario y al tener un búho tienes mas compañía y puedes rolear mas.

## Images of changes
Garronte en la oficina del chaplain: 

![Buho6](https://user-images.githubusercontent.com/61878952/76172776-717b0680-6199-11ea-985b-5f6cb5b29018.PNG)

Garronte de frente: 

![Buho1](https://user-images.githubusercontent.com/61878952/76172791-92dbf280-6199-11ea-802e-54062cf45bc6.PNG)

Garronte de espaldas: 

![Buho2](https://user-images.githubusercontent.com/61878952/76172799-a0917800-6199-11ea-8da0-c2ec70fce930.PNG)

Garronte de los lados: 

![Buho3](https://user-images.githubusercontent.com/61878952/76172803-aedf9400-6199-11ea-8440-4eae08d303ef.PNG) 

![Buho4](https://user-images.githubusercontent.com/61878952/76172807-b141ee00-6199-11ea-9521-4a41f167a03d.PNG)

Garronte muerto: 

![Buho5](https://user-images.githubusercontent.com/61878952/76172833-dd5d6f00-6199-11ea-8984-73608929f605.PNG)

## Changelog
:cl:
add: Añade un búho como la mascota del chaplain.
add: Añade una cama para la nueva mascota.
add: Añade un nuevo emote (*hoot) con su sonido propio.
/:cl:

Jinouga19 committed:
Buho_v0.1

Buho de prueba para ser la mascota del chaplain, aun me queda editar varias cosas y los sprites

Jinouga19 committed:
Buho_v0.2

No se si esto lo arregla un poco

Jinouga19 committed:
Update owl.dm

Jinouga19 committed:
Buho_v0.4

Ahora me parece que no te puedes meter dentro de el

Jinouga19 committed:
Buho_v0.5

Sprites cambiados y mejorados

Jinouga19 committed:
Buho_v0.6

Ahora tiene su propia cama.

Jinouga19 committed:
Buho_v0.6

Ahora el buho esta implementado en el mapa y tiene su propia cama

Jinouga19 committed:
Buho_casi_terminado

Jinouga19 committed:
Metiendo_mapmergue

Jinouga19 committed:
Buho_implementado_mapmergue

Jinouga19 committed:
Update owl.dm

Para ver si puedo usarlo como  NPC respawn

Jinouga19 committed:
Update hispania.dmm

Jinouga19 committed:
Buho implementado

Spircen commented on _maps/arena_template.dmm.backup:
```diff
@@ -0,0 +1,27 @@
+"a" = (/turf/unsimulated/wall,/area/space)
```
Este no hay que enviarlo uwu

Spircen replied:
Tampoco envíes el de _maps/map_backups/MinaGate.dmm.backup que no me deja comentar.

Jinouga19 replied:
Vale, lo quito entonces no?

Spircen replied:
Sip, borra los dos archivos del PR. A la par que ahorita vemos como hacemos lo de que map mergue no te funciona.

Silverexperience commented on code/HISPANIA/game/objects/structures/stool_bed_chair_nest/bed.dm:
```diff
@@ -28,3 +28,8 @@
 	anchored = TRUE
+
+/obj/structure/bed/dogbed/garronte
+	desc = "A comfy-looking duck bed. You can even strap your pet in, in case the gravity turns off."
```
Dice que es una cama comoda de pato. ¿No deberia ser de búho?

Jinouga19 replied:
Habia cogido la descripcion del de la cama de Duck. Creia que lo habia cambiado, gracias por avisar

Silverexperience replied:
Tambien en los emotes pone suspirar, no se si lo quieres asi o es por coger el mob de duck.

Jinouga19 replied:
No, ese lo quería así. Me pareció gracioso

Jinouga19 committed:
Codigo_cambio_nombre_cama_descripcion

La descripcion de la cama se ha cambiado

Jinouga19 committed:
Delete MinaGate.dmm.backup

Jinouga19 committed:
Delete arena_template.dmm.backup

Jinouga19 commented:
¿Sabéis como se puede hacer para que el búho sea controlado por fantasmas.? Como runtime, Ian, etc...

Spircen commented on code/HISPANIA/modules/mob/living/simple_animal/friendly/owl.dm:
```diff
@@ -0,0 +1,19 @@
+	icon_state = "owl"
+	icon_living = "owl"
+	icon_dead = "owl_dead"
+	speak_emote = list("sighs")
```
Este mas el emote_hear y el emote_see hay que cambiarlos a un ruido especifico que hagan los buhos. No se que ruido hacen los buhos eso si.

Jinouga19 replied:
Mm, luego miro

Silverexperience replied:
Los búhos creo que hacen hoot.

Spircen replied:
HOOTERS

Jinouga19 committed:
Update hispania.dmm

Jinouga19 committed:
Merge branch 'Buho' of https://github.com/Jinouga19/Paradise into Buho

Jinouga19 committed:
Mapmerge_funciona

Funciona la implementacion en el mapa bien

Jinouga19 committed:
Onomatopeya

Ahora ya habla como deberia

Jinouga19 committed:
Sonidos_buho

Ahora puede hablar!

Spircen commented on code/HISPANIA/modules/mob/living/simple_animal/friendly/owl.dm:
```diff
@@ -0,0 +1,19 @@
+	icon_state = "owl"
+	icon_living = "owl"
+	icon_dead = "owl_dead"
+	speak = list("Hoot.","Hoot?","Hoot!.","Cough"
```
Te falto cerrar esta lista.

Spircen replied:
Y solamente con el hoot es suficiente, no lo entenderá la gente. Esta parte esta de mas.

Jinouga19 committed:
Sonidos cambiados

Evankhell561 commented on code/HISPANIA/modules/mob/living/simple_animal/friendly/owl.dm:
```diff
@@ -0,0 +1,19 @@
+	icon_state = "owl"
+	icon_living = "owl"
+	icon_dead = "owl_dead"
+	speak = list("Hoot.","Hoot?","Hoot!.","Cough")
```
no pongas un punto luego de !
y pon el punto en la ultima "Cough."

Jinouga19 replied:
gracias, ya lo he cambiado

Jinouga19 committed:
Version casi final del buho

Ahora ya se puede coger como fantasma. Te dropea carne si le destripas, emote personalizado *hoot (si lo usas suena un archivo de audio .ogg en el que se oye la voz de un buho real) Ya me queda poco que mirar, mañana le pego un vistazo final porque creo que voy a añadir una cosa mas pequeña

Jinouga19 committed:
error corregido de hablar buho

Jinouga19 commented:
Porque me la han cerrado? No les ha gustado?

Jinouga19 commented:
No voy a cambiar nada mas, ya estaría terminado con todos los cambios hechos.

Jinouga19 commented:
Porque dice que hay conflictos en icons/hispania/mob/animals.dmi?

DanaDririon commented:
Porque hubo otro PR que hizo cambios en ese archivo, entonces tu PR no tiene esos cambios, provocando conflictos

Jinouga19 commented:
Entonces que hago, deberia cambiarlo?

Jinouga19 committed:
Update animals.dmi

Jinouga19 committed:
Daba un error y he vuelto a copiarlo

Deberia estar arreglado...

Jinouga19 committed:
actualizando .dmi

Jinouga19 committed:
Buho metido en .dmi

Jinouga19 commented:
Cierro el PR y lo abro en uno nuevo ya que he tenido problemas al sincronizar el master.... ( no lo tenia bien sincronizado y no se me updateaba) siento las molestias.
